backend/src/test/java/com/woowacourse/kkogkkog/application/CouponServiceTest.java

```diff
@@ -48,13 +50,13 @@ void setUp() {
         memberRepository.save(ARTHUR);
     }
 
-    @DisplayName("단일 쿠폰을 조회할 수 있다.")
     @Nested
-    class FindByIdTest {
+    @DisplayName("findById 메서드는")
+    class FindById {
 
-        @DisplayName("존재하는 쿠폰을 조회하는 경우 성공한다.")
         @Test
-        void findById() {
+        @DisplayName("쿠폰이 존재하면, 쿠폰 응답을 반환한다.")
+        void success() {
```

Comment:
"쿠폰 응답"이라는게 DTO를 직역한 것으로 느껴지네요. 보다 더 의미를 추가할 수 있으면 좋을 것 같아요.

```suggestion
        @DisplayName("쿠폰이 존재하면, 해당 쿠폰의 정보를 반환한다.")
        void success() {
```